### PR TITLE
cherry pick for 6.3

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
- Docs updates
- CHANGELOG update
- Bug fixes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,55 +1,65 @@
-<p align="center"><img width="70%" src="docs/data/rocAL_logo.png" /></p>
+# Changelog for rocAL
 
-# Changelog
+Full documentation for rocLibrary is available at [https://rocm.docs.amd.com/projects/rocAL/](https://rocm.docs.amd.com/projects/rocAL/en/latest/).
 
-## Online Documentation
+## rocAL 2.2.0 (unreleased)
```
Comment (spolifroni-amd) [inline CHANGELOG.md:5]:
```suggestion
## (Unreleased) rocAL 2.2.0 
```

**Review by spolifroni-amd — APPROVED** [review]
The change to (unreleased) needs to be made otherwise that section might get picked up by the 6.3 release notes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
 
-### Removals
-* rocdecode dependencies for package install removed
+* rocdecode installation disabled when running the setup script.
+
+### Removed
+
+* rocDecode dependencies for package install has been removed.
 
 ### Optimizations
```
Comment (spolifroni-amd) [inline CHANGELOG.md:46]:
```suggestion
### Optimized
```

**Review by spolifroni-amd — COMMENTED** [review]
Optimizations -> Optimized.

**Comment by LakshmiKumar23** [comment]
@kiritigowda to create a new cherry pick branch

**Pull request closed without merging**